### PR TITLE
Test write geom

### PR DESCRIPTION
**Pull request by kkiesling**
Tests for just the function `write_geometry()` in `isg.py`. Two test- one to capture normal behavior, and another to capture the warning.

### CODE REVIEW

File: tests/test_isg.py [hunk 1]
```diff
@@ -472,7 +473,42 @@ def test_set_tags(tagname, tagval, expname, expval, exptype):
 
 
 def test_write_geometry():
```
Comment (bam241) [inline tests/test_isg.py:475]: comment to explain what the test is about.

File: tests/test_isg.py [hunk 2]
```diff
@@ -472,7 +473,42 @@ def test_set_tags(tagname, tagval, expname, expval, exptype):
...
-    pass
+    # write a file
+    ig = isg.IsGm()
+    sname = 'write-test.h5m'
+    ig.write_geometry(sname, test_dir)
+    # check that file exists
+    exp_file = test_dir + '/' + sname
+    if isfile(exp_file):
+        r0 = True
```
Comment (bam241) [inline tests/test_isg.py:483]: I would initialize `r0` to `False`

and maybe `r0` is not a proper name...

**Review by bam241 — APPROVED** [review]
looking good.
some few comments, nothing major..

File: tests/test_isg.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 """tests for the IsoSurfGeom module"""
 from os import listdir, remove, getcwd, mkdir
 from os.path import isfile, isdir, join
+import shutil
```
Comment (bam241) [inline tests/test_isg.py:4]: not used.

File: tests/test_isg.py (hunk 2 above)
Comment (kkiesling) [reply tests/test_isg.py:483]: `r` is the common name of the True/False results for the asserts used in all the tests so I am going to leave it as `r`.

**Comment by kkiesling** [comment]
suggested changes have been addressed + added more robust check for the warnings

**Review by bam241 — APPROVED** [review]
Thx @kkiesling 

LGTM!

**Pull request merged**